### PR TITLE
Allow recovery code to be disabled

### PR DESCRIPTION
**Pull request by louischan-oursky**
ref #1529

### CODE REVIEW

**Review by carmenlau — COMMENTED** [review]
Screenshot 2022-06-08 at 12 15 38 PM

In the recovery code listing page, we may want to handle when the user doesn't have the recovery code yet. Maybe showing different copywriting?

Steps to reproduce:
- Disable recovery code
- User signup with mfa
- App enable recovery code
- User visit recovery code listing page

**Comment by louischan-oursky** [comment]
Perhaps just say you dont have any recovery yet, and just show the get new code button?

**Comment by louischan-oursky** [comment]
Screenshot 2022-06-09 at 1 38 01 PM

File: resources/authgear/templates/en/translation.json [hunk 1]
```diff
@@ -88,6 +88,7 @@
   "copy-button-label": "Copy",
   "copied-button-label": "Copied",
   "get-new-code-button-label": "Get new code",
```
Comment (carmenlau) [inline resources/authgear/templates/en/translation.json:90]: Seems it is replaced by `get-new-recovery-codes-button-label` and it is not used. Remove it?

**Pull request merged**